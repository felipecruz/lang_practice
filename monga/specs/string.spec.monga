char[] func () {
    string = "valid string";
    string = "valid string with \n like break";
    string = "valid string with \t tab";
    string = "valid string with \"";

    return string;
}
