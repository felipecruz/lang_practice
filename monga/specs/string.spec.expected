
Func Decl - Return Type: Char[] 
          - Paramenters 
   Assignment Var: string <- EXP: Const string: "valid string"
   Assignment Var: string <- EXP: Const string: "valid string with \n "
   Assignment Var: string <- EXP: Const string: "valid string with \\ "
   Assignment Var: string <- EXP: Const string: "valid string with \t "
   Assignment Var: string <- EXP: Const string: "valid string with \""
   Return EXP: Var: string 

